front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -60,6 +60,24 @@ import { defaultAdminCardPadding } from 'utils/styleConstants';
 
 import useContainerWidthAndHeight from 'hooks/useContainerWidthAndHeight';
 
+function roundToNearestMultipleOfFive(date: Date): Date {
+  const minutes = date.getMinutes();
+  const roundedMinutes = Math.ceil(minutes / 5) * 5;
+  return new Date(
+    date.getFullYear(),
+    date.getMonth(),
+    date.getDate(),
+    date.getHours(),
+    roundedMinutes
+  );
+}
+
+function calculateRoundedEndDate(startDate: Date, minutes = 30): Date {
+  const endDate = new Date(startDate);
+  endDate.setMinutes(startDate.getMinutes() + minutes);
+  return endDate;
+}
+
```

Comment:
Could you add these in a util file so they're more visible if we ever decide to allow event creation outside of projects? + also add some tests.